CRM/Report/Form/Event/Income.php

```diff
@@ -216,6 +208,7 @@ public function buildEventReport($eventIDs) {
 
     $statusDAO = $this->executeReportQuery($status);
 
+    $participantStatus = CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label");
```

Comment:
How about using `$participantStatus = $this->getActiveParticipantStatuses();`

Reply:
updated per below